SpatialGDK/Documentation/content/helper-scripts.md

```diff
@@ -2,8 +2,8 @@
 
 # Helper scripts
 
+These scripts are located in your UnrealGDK repo under `SpatialGDK\Build\Scripts\`
+
 | Helper script | Parameters | Description |
 | --- | --- | --- |
-| `Game\Scripts\Codegen.bat` | None | Generates SpatialOS C++ Worker code from the generated schema and type-bindings. |
-| `Game\Scripts\BuildWorkerConfig.bat` | None | Generates launch configurations for the SpatialOS Runtime. You need to re-run this if you've made changes to either of these files: </br> - `spatialos.UnrealClient.worker.json`</br> - `spatialos.UnrealWorker.worker.json` </br></br>You can find the generated launch configuration(s) in `<ProjectName>\spatial\build\assembly\<WorkerName>\*`|
-| `Game\Scripts\BuildWorker.bat` | `<target> <platform> <configuration> <YourGame>.uproject [--skip-codegen]` | Example: </br> `Game\Scripts\BuildWorker.bat ExampleGameEditor Win64 Development ExampleGame.uproject` </br></br> Build, cook and zip your Unreal server-workers and client-workers for use with a SpatialOS cloud deployment (uploaded using [`spatial cloud upload`](https://docs.improbable.io/reference/latest/shared/deploy/deploy-cloud)). </br></br>  The following `<target>`s  generate zipped workers: </br> - `<YourGame>` </br> - `<YourGame>Server` </br> - `<YourGame>Editor` </br></br> Any other `<target>` passes all arguments to `Engine\Build\BatchFiles\BuildWorker.bat` with no cooking or zipping performed.|
+| `BuildWorker.bat` | `<target> <platform> <configuration> <YourGame>.uproject [--skip-codegen]` | Example: </br> `BuildWorker.bat ExampleGameEditor Win64 Development ExampleGame.uproject` </br></br> Build, cook and zip your Unreal server-workers and client-workers for use with a SpatialOS cloud deployment (uploaded using [`spatial cloud upload`](https://docs.improbable.io/reference/latest/shared/deploy/deploy-cloud)). </br></br>  The following `<target>`s  generate zipped workers: </br> - `<YourGame>` </br> - `<YourGame>Server` </br> - `<YourGame>Editor` </br></br> Any other `<target>` passes all arguments to `Engine\Build\BatchFiles\Build.bat` with no cooking or zipping performed.|
```

Comment:
Maybe we also want to add `Setup.bat` here too?

Reply:
This section just describes how to use the script, not it's dependencies.